ideascube/configuration/migrations/0002_settings_to_configuration.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import Setting
```

Comment:
Crossed answers: I think adding the new confing system AND having a data migration AND having a migration to delete the Setting model is the right thing to do.